### DISCUSSION TITLE
Water balance components of CWatM - Water use integartion in the model

### DISCUSSION THREAD

**Discussion by ajawadh** [iiasa/CWatM#106] [Q&A] [answered]
Dear CWatM community,

I am currently using the CWatM to conduct a global hydrological analysis of the model's various outputs. I have the following question on the model components:

1- Do the water demand components affect the total runoff component or is the total runoff correspond to the physical components of the water cycle? In the CWatM article, it states that water demand is abstracted hierarchically starting from renewable groundwater, if not satisfied, then from surface bodies, if not satisfied, then from non-renewable groundwater resources. Are these abstractions applied separately without directly changing the total runoff volume?
2- What constitutes the total water storage (TWS) output in CWatM? Can we say it is the (Groundwater storage) + (Reservoir storage) + (Lake storage) + (River storage). Am I missing some components to infer the TWS? 

Thanks and regards

**Comment by mikhailsmilovic** [comment, marked as answer]
Guten Morgen Awad, great questions (and answer clarification)!

1: Water abstractions are performed when calculating discharge. Water demand is abstracted from availability in the following order: channel storage, lakes, reservoirs, renewable groundwater, and fossil groundwater. A previous post about how downstream flow is affected: https://github.com/iiasa/CWatM/discussions/62#discussioncomment-5194275

2: Total Water Storage (TWS) is discussed here: https://github.com/iiasa/CWatM/discussions/32#discussioncomment-2971901. Beyond channels, lakes & reservoirs, and groundwater, tws includes totalSto, which is soil moisture in the three layers, interception storage, and snow storage, and then another storage component called small lakes & reservoirs. 

If the interest is to compare with GRACE satellites for changes in total water storage, this [video](https://youtu.be/aWQa0ZdYaWY?si=7DXLm9NZeq3UI8ky) goes through preparing total water storage outputs to compare with GRACE. An update from the video, we now suggest generally for limitAbstraction = False to allow for fossil water abstractions.

**Reply by ajawadh** [reply]
Good morning Smilovic,

Thank you for your reply and your references. I've went through the referenced comments and I have a comment on your answer #62. In #62 , you have provided the following equation: "Qgrid = runoff - channel evaporation - channel abstraction + return flow (all grid cell specific)." If I understood this equation correctly, this implies that water abstractions are performed and are embedded only when calculating the dischrage variable of each gird. However, the runoff generation variable itself (runoff) is not affected by abstraction.  Did I understood it correctly?

**Reply by mikhailsmilovic** [reply]
Guten Morgen Awad, yes, you are correct. I will edit my answer to be clearer. Thank you!

**Reply by ajawadh** [reply]
Thank you for quick feedback!
